### DISCUSSION TITLE
How to use dev tools in production?

### DISCUSSION THREAD

**Discussion by vipinnair22** [TanStack/query#1821] [General]
I'm trying to enable dev tools in production mode based on a flag. I found that there's a min.js file that could be used.
https://unpkg.com/browse/react-query@3.3.0/dist/react-query-devtools.production.min.js

I'm trying to figure out how to import this file and render the dev tools component. Any help is appreciated. Thanks.

**Comment by sunglee** [comment]
Did you find a solution?

**Comment by kostia1st** [comment]
Just import the devtools like this
```ts
import { ReactQueryDevtools } from 'react-query/devtools/development';
```

Then they'll render on all (DEV or PROD) environments the same way.
